data/accubits.mdx

```diff
@@ -0,0 +1,19 @@
+---
+name: Accubits Technologies Pvt. Limited
+description: Accubits Technologies is an enterprise solutions development company focusing on AI and Blockchain technologies, based in Virginia, USA with offices in India and UAE. Founded in 2012, Accubits was created to leverage technology to empower creative minds all over the world. Pioneered as a custom Blockchain solutions provider, Accubits is one of the oldest and most experienced Blockchain development houses in the market.
+founded: February 08, 2012
+based: Service
+domain: AI, Blockchain
+logo: accubits_logo.png
+location: Trivandrum
+address: 5th floor & 8th floor, Yamuna Building, Technopark Phase 3, Trivandrum, Kerala, 695581.
+technologies: 
```

Comment:
Technologies should include the specific modules, programming languages.etc that is used by the company and not the domains the company works in.